tools/aa/commands/InitCommand.java

```diff
@@ -56,17 +62,17 @@ public InitCommand(FileUtils fileUtils, GitRepoFactory gitRepoFactory) {
   public boolean run(String[] args) {
     // TODO: Add Flags.parse() support for specifying a particular class, not a whole package
     Flags.parse(args, InitCommand.class.getPackage());
-
     try {
       if (!fileUtils.folderEmptyOrNotExists(basePath.get())) {
         System.out.println("Error: Base folder exists and is not empty");
         System.exit(1);
       }
+      wasBaseFolderExisted = fileUtils.folderExists(basePath.get());
       // Create folders
-      fileUtils.mkdirs(fileUtils.joinPaths(basePath.get(), "head"));
-      fileUtils.mkdirs(fileUtils.joinPaths(basePath.get(), "ws"));
-      fileUtils.mkdirs(fileUtils.joinPaths(basePath.get(), "local"));
-      fileUtils.mkdirs(fileUtils.joinPaths(basePath.get(), "logs"));
+      fileUtils.mkdirs(fileUtils.joinPaths(basePath.get(), HEAD_FOLDERNAME));
```

Comment:
For strings used only once, it's mostly ok to not create a `static final`, since it's easier to read it when the string is there.

Reply:
Done.